lib/rbac/filterer.rb

```diff
@@ -445,7 +445,12 @@ def scope_targets(klass, scope, rbac_filters, user, miq_group)
         # if subclasses of MetricRollup or Metric, use the associated
         # model to derive permissions from
         associated_class = rbac_class(scope)
+        if associated_class.respond_to?(:scope_by_tenant?) && associated_class.scope_by_tenant?
+          scope_associated_ids = scope_to_tenant(associated_class, user, miq_group).ids
+        end
+
         filtered_ids = calc_filtered_ids(associated_class, rbac_filters, user, miq_group)
+        filtered_ids = [scope_associated_ids, filtered_ids].compact.flatten
```

Comment:
please pass this scope into `filtered_ids`.

Trying hard to keep that set logic in there so we can move to the database in the future.

(all but one of those params are currently scopes)